src/get-started/java-binding.md

```diff
@@ -41,25 +30,42 @@ You can also install Exonum Java App manually:
   unzip /path/to/downloaded/exonum-java-0.6.0-release.zip
   ```
 
-
-### Linux
-
-- Install [Libsodium][libsodium] development library for your system.
-  You can use the following command for Ubuntu:
-
+- Install necessary runtime dependencies: [Libsodium][libsodium] and
+  RocksDB libraries (the latter is only needed on Mac OS):
+  
+  __Linux (Ubuntu)__
```

Comment:
Does it turn into tabs :crossed_fingers: ?

Reply:
Oh, actually it did not, but I think the latest commit should work

Reply:
I got confused by the double underscores — but do two consecutive examples work as _tabs_:

![image](https://user-images.githubusercontent.com/7531824/59337214-c708ac80-8d08-11e9-973c-d984ac731156.png)

Reply:
No, they would work as spoilers (see example [here](https://exonum.com/doc/version/latest/advanced/node-management/#response_4))